charts/tekton-common/templates/secrets-slack.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: secrets-slack
```

Comment:
maybe name `slack-secrets` to match with line 11

Reply:
changed!